[2020-08-07 22:41:59,698] | {{taskinstance.py:616}} | INFO | Dependencies all met for <TaskInstance: store_DAG.check_file_exists 2020-08-04T00:00:00+00:00 [queued]>
[2020-08-07 22:41:59,811] | {{taskinstance.py:616}} | INFO | Dependencies all met for <TaskInstance: store_DAG.check_file_exists 2020-08-04T00:00:00+00:00 [queued]>
[2020-08-07 22:41:59,814] | {{taskinstance.py:834}} | INFO | 
--------------------------------------------------------------------------------
[2020-08-07 22:41:59,818] | {{taskinstance.py:835}} | INFO | Starting attempt 1 of 3
[2020-08-07 22:41:59,822] | {{taskinstance.py:836}} | INFO | 
--------------------------------------------------------------------------------
[2020-08-07 22:42:00,020] | {{taskinstance.py:855}} | INFO | Executing <Task(BashOperator): check_file_exists> on 2020-08-04T00:00:00+00:00
[2020-08-07 22:42:00,024] | {{base_task_runner.py:133}} | INFO | Running: ['airflow', 'run', 'store_DAG', 'check_file_exists', '2020-08-04T00:00:00+00:00', '--job_id', '14', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmp03c91m6c']
[2020-08-07 22:42:16,331] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists [2020-08-07 22:42:16,330] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=4269
[2020-08-07 22:42:16,617] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2020-08-07 22:42:16,618] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists   """)
[2020-08-07 22:42:18,902] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists [2020-08-07 22:42:18,901] {{__init__.py:51}} INFO - Using executor LocalExecutor
[2020-08-07 22:42:24,328] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists [2020-08-07 22:42:24,319] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
[2020-08-07 22:42:24,869] | {{base_task_runner.py:115}} | INFO | Job 14: Subtask check_file_exists [2020-08-07 22:42:24,865] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.check_file_exists 2020-08-04T00:00:00+00:00 [running]> on host 5f4c3a456296
[2020-08-07 22:42:25,132] | {{bash_operator.py:81}} | INFO | Tmp dir root location: 
 /tmp
[2020-08-07 22:42:25,137] | {{bash_operator.py:91}} | INFO | Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=store_DAG
AIRFLOW_CTX_TASK_ID=check_file_exists
AIRFLOW_CTX_EXECUTION_DATE=2020-08-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-04T00:00:00+00:00
[2020-08-07 22:42:25,147] | {{bash_operator.py:105}} | INFO | Temporary script location: /tmp/airflowtmp5khi65j5/check_file_existssgt8dcw6
[2020-08-07 22:42:25,148] | {{bash_operator.py:115}} | INFO | Running command: shasum ~/store_files_airflow/raw_store_transactions.csv
[2020-08-07 22:42:25,265] | {{bash_operator.py:124}} | INFO | Output:
[2020-08-07 22:42:25,903] | {{bash_operator.py:128}} | INFO | e9e298e9629d72f3255b4838ba07bd220150a146  /usr/local/airflow/store_files_airflow/raw_store_transactions.csv
[2020-08-07 22:42:25,916] | {{bash_operator.py:132}} | INFO | Command exited with return code 0
[2020-08-07 22:42:29,595] | {{logging_mixin.py:95}} | INFO | [[34m2020-08-07 22:42:29,577[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 0[0m
